docs/topics/con_about-messaging-work-queue.adoc

```diff
@@ -0,0 +1,14 @@
+[id='about-messaging-work-queue_{context}']
+= About Shared Messaging Work Queue
```

Comment:
Remove "Shared".

Reply:
Removed, and replaced the hard-coded name with an attribute to prevent similar blunders:) Thank you.